### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by todi1856**
Bug fixes
- Fix NullReference exceptions after domain reload
- Fix Stacktrace window not able to load when active platform is not Android
- Fix issue where Open terminal didn't work on Windows for some reason
- Fix issue where connect to IP would lock editor, it's now multithreaded
- Reworked connect to ip window, you can now query ip from devices connected via USB, making your life easier 
ip window
- Added icons for messages
- Split message colors ProSkin vs FreeSkin
Icons

- https://github.com/Unity-Technologies/com.unity.mobile.logcat/issues/54 put Android Logcat Settings under Analysis
- https://github.com/Unity-Technologies/com.unity.mobile.logcat/issues/53 don't draw icons if font is too small
- https://github.com/Unity-Technologies/com.unity.mobile.logcat/issues/52 added disconnect button
- https://github.com/Unity-Technologies/com.unity.mobile.logcat/issues/51 right clicking will not deselect items, if you're right clicking on selected item
- https://github.com/Unity-Technologies/com.unity.mobile.logcat/issues/49 correctly parse android version 9

### CODE REVIEW

File: com.unity.mobile.android-logcat/Editor/AndroidLogcatStacktraceWindow.cs [hunk 1]
```diff
@@ -13,6 +12,7 @@ namespace Unity.Android.Logcat
 {
     internal class AndroidLogcatStacktraceWindow : EditorWindow
     {
+#if PLATFORM_ANDROID
```
Comment (vincent-unity) [inline com.unity.mobile.android-logcat/Editor/AndroidLogcatStacktraceWindow.cs]: Weird to me. Do we have the UI to open this window if current active platform is not android?

File: com.unity.mobile.android-logcat/Editor/AndroidLogcatIPWindow.cs [hunk 2]
```diff
@@ -7,49 +8,119 @@ namespace Unity.Android.Logcat
...
+                    var name = i < m_ConnectedDeviceDetails.Length && !string.IsNullOrEmpty(m_ConnectedDeviceDetails[i])
+                        ? m_ConnectedDeviceDetails[i]
+                        : m_ConnectedDevices[i];
+
+
+                    EditorGUILayout.LabelField(name, EditorStyles.label);
+
+                    if (GUILayout.Button(" Copy IP ", GUILayout.ExpandWidth(false)))
+                    {
+                        m_IpString = CopyIP(m_ConnectedDevices[i]);
```
Comment (vincent-unity) [inline com.unity.mobile.android-logcat/Editor/AndroidLogcatIPWindow.cs]: Probably be good to copy to `GUIUtility.systemCopyBuffer`? Just considered from UX aspect. :)

File: com.unity.mobile.android-logcat/Editor/AndroidLogcatStacktraceWindow.cs (hunk 1 above)
Comment (vincent-unity) [reply com.unity.mobile.android-logcat/Editor/AndroidLogcatStacktraceWindow.cs]: Oh, just noticed bug https://fogbugz.unity3d.com/f/cases/1180263/ ...

File: com.unity.mobile.android-logcat/Editor/AndroidLogcatIPWindow.cs (hunk 2 above)
Comment (todi1856) [reply com.unity.mobile.android-logcat/Editor/AndroidLogcatIPWindow.cs]: yes, good idea

Comment (todi1856) [reply com.unity.mobile.android-logcat/Editor/AndroidLogcatIPWindow.cs]: Fixed

**Pull request merged**